doc/api/cli.md

```diff
@@ -361,6 +361,15 @@ reference. Code may break under this flag.
 `--require` runs prior to freezing intrinsics in order to allow polyfills to
 be added.
 
+### `--force-node-api-uncaught-exceptions-policy=bool`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enforces 'uncaughtException' event on Node-API asynchronous callbacks.
+
+Allows value of `true` and `false`.
```

Comment:
A bool flag is a bit odd. Which is the default behavior? If the default is `false`, then I would expect that just using `--force-node-api-uncaught-exceptions` would imply `true`. Otherwise if the default is `true`, then I would expect a flag like `--dont-force-node-api-uncaught-exceptions`.